docs/source/random_cat.md

```diff
@@ -68,25 +73,47 @@ To make sure everything went well, check the linked .fits files with
 canfar_avail_results -i tile_numbers.txt --input_path output/run_sp_combined_flag/mask_runner/output -x fits -v -m
 ```
 
-## Create random catalogue
+## Create random catalogue and helapix mask per tile
 
-First, create a random catalogue for each input tile and mask file,
+Run
 ```bash
 shapepipe_run -c $SP_CONFIG/config_Rc.ini
 ```
-Next, merge those catalogues into a numpy binary (``.npy``) file,
+The random catalogue and, with the config entry `SAVE_MASK_AS_HEALPIX = True``
```

Comment:
is there an extra ` here or was that intentional

Reply:
Ah yes, there is a typo. We need two ` for quotation in markdown. It's fixed.